src/launcher/BrowserRunner.ts

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright 2020 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as debug from 'debug';
+
+import * as removeFolder from 'rimraf';
+import * as childProcess from 'child_process';
+import { helper, assert, debugError } from '../helper';
+import type { LaunchOptions } from './LaunchOptions';
+import { Connection } from '../Connection';
+import { WebSocketTransport } from '../WebSocketTransport';
+import { PipeTransport } from '../PipeTransport';
+import * as readline from 'readline';
+import { TimeoutError } from '../Errors';
+
+const removeFolderAsync = helper.promisify(removeFolder);
+const debugLauncher = debug('puppeteer:launcher');
+
+export class BrowserRunner {
```

Comment:
the functionality and contents of this class has not changed, other than the `private` keyword on the instance properties.